OracleLinuxDevelopers/oraclelinux9/haproxy/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+FROM ghcr.io/oracle/oraclelinux:9
+
+RUN \
+    dnf -y install --setopt=tsflags=nodocs haproxy && \
```

Comment:
Same as OL8: please remove the `setopt` here.

Reply:
did the same for OL9